apps/backend/src/datasources/mockups/AdminDataSource.ts

```diff
@@ -41,6 +41,7 @@ export class AdminDataSourceMock implements AdminDataSource {
       new Settings(SettingsId.FEEDBACK_MAX_REQUESTS, '', '2'),
       new Settings(SettingsId.FEEDBACK_EXHAUST_DAYS, '', '90'),
       new Settings(SettingsId.FEEDBACK_FREQUENCY_DAYS, '', '14'),
+      new Settings(SettingsId.DEFAULT_NUM_REVIEWS_REQUIRED, '', '2'),
```

Comment:
Maybe including `FAP` keyword here is beneficial as well because it is not clear what reviews is this for.